src/script.js

```diff
@@ -182,6 +184,13 @@ export function getPartnerAttributionID() : ?string {
     return getSDKAttribute(SDK_SETTINGS.PARTNER_ATTRIBUTION_ID);
 }
 
+export function getPageType() : ?string {
+    const pageType = getSDKAttribute(SDK_SETTINGS.PAGE_TYPE);
+    const validPageType = pageType ? PAGE_TYPES[pageType.toUpperCase()] : false;
+
+    return validPageType ? pageType : '';
```

Comment:
I was wondering, do we want to force the pageType to always be uppercase? For example, if page type is passed in as lowercase as 'home' (`'data-page-type': 'home'`), should we log it as 'HOME'? If so we'd need to tweak this. Ex:

```suggestion
    return validPageType ? pageType.toUpperCase() : '';
```

Reply:
Yes.  If you look at the `PAGE_TYPES` enum, they are all upper case.  So the customer can pass whatever and we convert it to upper case to match the key.

Reply:
I see that. I guess my question is, should `getPageType()` always return as uppercase? Do we always want them logged as uppercase for consistency? I see in the unit tests that it currently returns as lowercase 'home' if that's what is passed in as the data attribute. So the end user can technically pass in either "home" or "HOME", both and are valid with the ENUM check.